src/index.js

```diff
@@ -1,5 +1,5 @@
 // Test import of a JavaScript function
-import {example} from './js/example'
+import example from './js/example'
```

Comment:
Is there a reason for changing this to default? Not using default exports helps reduce bugs: https://palantir.github.io/tslint/rules/no-default-export/ But it's not a huge deal.

Reply:
Addressed in commit [d633e41](https://github.com/taniarascia/webpack-boilerplate/pull/40/commits/d633e41be30483029eebc44668953f39d199b711).